helperAPI.py

```diff
@@ -505,6 +512,24 @@ def getDriver(DOCKER=False):
                 service=EdgeService(),
                 options=options,
             )
+            """
+             #! What I added
+            chrome_options = Options()
+            chrome_options.add_argument('--ignore-certificate-errors')
+            chrome_options.add_argument('--ignore-ssl-errors')
```

Comment:
I dont think you should ignore cert and ssl errors when you are giving usernames and passwords to these sites